### PR TITLE
MB-10877 SIT Allowance is editable by GHC API allowances handlers

### PR DESCRIPTION
**Pull request by Ryan-Koch**
## Summary
This makes it so the GHCAPI handlers for the TOO and the Services Counselor for editing allowances are able to edit the SIT allowance value on the entitlement table.

There will be follow up to migrate over to this value from the shipment based one in a couple of future PRs. The reason for this is that while the SIT entitlement is something that applies to the shipment it is always the same for every shipment on the same move. 

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

### CODE REVIEW

File: scripts/gen-server [hunk 1]
```diff
@@ -24,7 +24,7 @@ swagger_func() {
   if [ -n "${CIRCLECI+x}" ]; then
     # In CircleCI the docker image has the binary installed
     echo "RUNNING IN CIRCLECI"
-    /usr/local/bin/swagger "$@"
```
Comment (leia-sefkin) [inline scripts/gen-server:27]: did you mean to commit this change?

**Review by leia-sefkin — CHANGES_REQUESTED** [review]
LGTM except there appears to be an errant commit to scripts/gen-server, once that's removed i'd say it's good to be merged

File: scripts/gen-server (hunk 1 above)
Comment (Ryan-Koch) [reply scripts/gen-server:27]: ack no I didn't. I'll remove that.

Comment (Ryan-Koch) [reply scripts/gen-server]: Just pushed the fix to put it back how it was.

**Review by leia-sefkin — APPROVED** [review]

**Pull request merged**